executor/infoschema_reader.go

```diff
@@ -2820,6 +2822,145 @@ func (e *memtableRetriever) setDataForRegionLabel(ctx sessionctx.Context) error
 	return nil
 }
 
+func (e *memtableRetriever) setDataFromPlacementRules(ctx context.Context, sctx sessionctx.Context, schemas []*model.DBInfo) error {
+	checker := privilege.GetPrivilegeManager(sctx)
+	is := sctx.GetInfoSchema().(infoschema.InfoSchema)
+	var rows [][]types.Datum
+
+	// Get global PLACEMENT POLICIES
+	// Currently no privileges needed for seeing global PLACEMENT POLICIES!
```

Comment:
This is a great question. We have not documented it yet, and there is not quite an equiverlant in MySQL since we went with global scope for usability vs. schema scope:

Since global privileges can be assigned to any table, I think almost has to be the case that no permissions are required.

If we do require a privilege, it would have to be very basic like `CREATE` or `ALTER` on any object. AFAIK the privilege API does not currently have the ability to check for "CREATE on any object" (which semantically is different from "CREATE on global").

The question is, is it problematic that a user with only `USAGE` can see placement policies? They could technically reveal topology information, which might be considered sensitive.

@dveeden what do you think here?

Reply:
I don't see any problem for exposing placement policies to users with only global `USAGE` (no privileges).